data/projects/chiquito.ts

```diff
@@ -0,0 +1,26 @@
+import { ProjectInterface } from "@/lib/types"
+
+const description = `
+Chiquito is a high-level structured language for implementing zero knowledge proof circuits, currently being implemented in the DSL Working Group of PSE. It is a state machine zk-language that provides better syntax and abstraction. It can automatically compiler arbitrary boolean expressions for state machine transitions. It can optimise the resulting arithmetization. Chiquito has a Halo2 backend, which is a low level zkDSL that writes circuits using the PLONKish arithmetization and is working on supporting additional backends. Chiquito circuits can be written using both Python and Rust. 
+`
+
+export const chiquito: ProjectInterface = {
```

Comment:
@leolara this needs to be added inside the `projects.ts` file to see the project in the list

Reply:
Done